app/templates/issues.hbs

```diff
@@ -21,14 +24,16 @@
     </tr>
   </thead>
   <tbody>
-    {{#each model as |issue|}}
+    {{#each this.model as |issue|}}
       <tr>
-        <td><a target="_blank" href={{issue.repositoryHtml}}>{{issue.repositoryName}}</a></td>
+        <td><a target="_blank" rel="noopener" href={{issue.repositoryHtml}}>{{issue.repositoryName}}</a></td>
         <td>
-          <a target="_blank" class="github-issue-link" href={{issue.htmlUrl}}>{{issue.title}}</a>
+          <a target="_blank" rel="noopener" class="github-issue-link" href={{issue.htmlUrl}}>{{issue.title}}</a>
 
           {{#each issue.labels as |lb|}}
-            <a onclick={{action (mut label) lb.name}}>{{github-label label=lb.name labelColor=lb.color}}</a>
+            <button class="github-label-button" {{action (mut this.label) lb.name}}>
```

Comment:
in `filter-test` (line 19) it looks for the second `a` tag but now it's a `button`